sql/src/main/java/io/crate/planner/consumer/UpdatePlanner.java

```diff
@@ -186,7 +188,14 @@ private static ExecutionPlan sysUpdate(PlannerContext plannerContext,
         Reference idReference = requireNonNull(tableInfo.getReference(DocSysColumns.ID), "Table must have a _id column");
         SysUpdateProjection updateProjection = new SysUpdateProjection(idReference.valueType(), assignmentByTargetCol);
         WhereClause where = new WhereClause(SubQueryAndParamBinder.convert(query, params, subQueryResults));
-        return createCollectAndMerge(plannerContext, tableInfo, idReference, updateProjection, where);
+        return createCollectAndMerge(plannerContext,
+                                     tableInfo,
+                                     idReference,
+                                     updateProjection,
+                                     where,
+                                     1,
+                                     1,
+                                     MergeCountProjection.INSTANCE);
```

Comment:
I am not so happy to juggle with 2 projection types around here. One solution could be to split UpdateProjection to 2 types one of for RowCount and one for ResultSets with the designated logic then.